docs/account-management/plans/index.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 10
+title: Plans
+---
+
+This section explains the differences between each Macrometa plan, and how to view and change your own plan.
+
+For more information about limits, refer to [Tenant Quotas and Limits](../../references/quotas.md)
+
+- **Playground -** A test environment to familiarize yourself with Macrometa GDN. Production use is disabled.
+- **Scale -** The standard production package. Pay only for what you need, and increase or decrease your limits as you go.
+- **Enterprise -** Remove limits from your account.
```

Comment:
@mikeridley Can you take a look at these three bullet points and let me know if this is an accurate description of the different Plans?  I'm still a little fuzzy on the exact differences between Scale and Enterprise, but the impression I get is that Scale has limits and Enterprise does not. 

Feel free to suggest alternate wording if you like.  Thanks!